db/schema.rb

```diff
@@ -32,5 +41,20 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "view_parties", force: :cascade do |t|
+    t.string "title"
+    t.string "date"
+    t.string "time"
+    t.integer "duration"
+    t.string "poster"
+    t.bigint "user_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_view_parties_on_user_id"
+  end
+
   add_foreign_key "friendships", "users"
+  add_foreign_key "party_guests", "users"
+  add_foreign_key "party_guests", "view_parties"
+  add_foreign_key "view_parties", "users"
 end
```

Comment:
renaming these resources and updating file names/method calls/object instantiations was painful